portality/formcontext/formcontext.py

```diff
@@ -676,13 +677,14 @@ def finalise(self):
 
         # if this application is being accepted, then do the conversion to a journal
         if self.target.application_status == constants.APPLICATION_STATUS_ACCEPTED:
+            # remember whether this was an update request or not
```

Comment:
unnecessary duplication

Reply:
meh, maybe.  is_update_request isn't required in all the if/else statements, so in some code paths this would be unnecessarily executed.  Six of one, half a dozen of the other.  I prefer it this way myself, just for clarity - each if/else branch is self-contained in its behaviour.

Reply:
Okay, we'll keep it as-is!